views/conditions/fungal-nail-infection.html

```diff
@@ -6,6 +6,10 @@
   <div id="local-header">
     <div class="header-local-container">
       <h1>{{ pagetitle }}</h1>
+      <div class="measure">
```

Comment:
Fair point, although where we're *not* using the grid structure (i.e. somewhere like this) perhaps some kind of "good measure" utility class is a handy tool?